app/reports/digital_serials_title_parts.rb

```diff
@@ -19,26 +20,37 @@ def initialize(filename:)
   def report
     raise "Input file missing: #{filename}" unless File.exist?(filename)
 
-    puts 'druid,catalogRecordId,part_names,part_numbers'
-    File.foreach(filename) do |raw_line|
+    puts 'druid,catalogRecordId,part_names,part_numbers,part_label'
+    File.foreach(filename) do |raw_line| # rubocop:disable Metrics/BlockLength
       druid = "druid:#{raw_line.chomp}"
```

Comment:
Totally trivial, admittedly, but:

```suggestion
    File.foreach(filename, chomp: true) do |bare_druid| # rubocop:disable Metrics/BlockLength
      druid = "druid:#{bare_druid}"
```